runtests.sh

```
@@ -137,8 +137,13 @@ function print_version {
 }
 
 function install_deps {
-    echo "Pip installing MONAI development dependencies and compile MONAI cpp extensions..."
-    ${cmdPrefix}"${PY_EXE}" -m pip install --no-build-isolation -r requirements-dev.txt
+    echo "Pip installing MONAI development dependencies..."
+    # needed for Python<3.11
+    ${cmdPrefix}"${PY_EXE}" -m pip install -U tomli
+    # create a temporary requirements file and install using it
+    REQ=$(mktemp --tmpdir XXX.txt)
+    ${cmdPrefix}"${PY_EXE}" monai/config/print_dependencies.py all testing > "$REQ"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)